enh(operatingsystems-as400-connector): add enable and start daemon CTOR-1714 (#4507)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/operatingsystems-as400-connector.md

update-alternatives --config java
```

> This instructions allows to started and enabled the daemon:

Reviewer: ```suggestion
Use the following instructions to start and enable the daemon:
```

```shell
systemctl status centreon-as400
```

> Returns :

Reviewer: ```suggestion
This command returns:
```

```shell
○ centreon-as400.service - Centreon AS400
     Loaded: loaded (/usr/lib/systemd/system/centreon-as400.service; disabled; preset: disabled)
     Active: inactive (dead)
```

> Then do this : 

Reviewer: ```suggestion
Then, execute the following commands: 
```
